### PR TITLE
doc: add more information about the logger setup/usage

### PR DESCRIPTION
**Pull request by tessig**

### CODE REVIEW

**Comment by tessig** [comment]
this includes docu for #412 and should wait for it

File: docs/1. Flamingo Basics/5. Flamingo Production.md [hunk 1]
```diff
@@ -47,8 +47,94 @@ CMD ["serve"]
...
+	flamingo.App([]dingo.Module{
+        // your modules
+	}, flamingo.WithCustomLogger(new(MyLogModule)))
+}
+```
+
+This Flamingo module must at least bind an implementation to the `flamingo.Logger` interface.
+
+An alternative implementation available in the core is `silentzap`. Silentzap wraps the zap logger and only logs when 
+an error happens, but then all levels in the same context/trace.
```
Comment (carstendietrich) [inline docs/1. Flamingo Basics/5. Flamingo Production.md:136]:
```suggestion
an error happens, but then all levels are in the same context/trace.
```

File: docs/1. Flamingo Basics/5. Flamingo Production.md [hunk 2]
```diff
@@ -47,8 +47,94 @@ CMD ["serve"]
...
+
+To enable the Trace log level, you need to compile your application with the `tracelog` build tag:
+
+```sh
+go build -tags tracelog
+```
+
+#### Logging with Context and additional fields
+
+Flamingo's logger also supports logging with context and additional fields, which is useful for adding additional metadata to your log entries:
```
Comment (carstendietrich) [inline docs/1. Flamingo Basics/5. Flamingo Production.md:100]:
```suggestion
Flamingo's logger also supports logging with context and additional fields, which is useful for adding extra metadata to your log entries:
```

**Review by carstendietrich — APPROVED** [review]
Fine for me only some minor nitpicks

**Pull request merged**